desktop/apps/consignments2/client/actions.js

```diff
@@ -46,6 +54,7 @@ export function addImageToUploadedImages (fileName, src) {
 export function chooseArtistAndAdvance (value) {
   return (dispatch) => {
     dispatch(updateArtistId(value._id))
+    dispatch(updateArtistName(value.name))
```

Comment:
The `artist_id` is actually nested under all the `inputs` part of the state, which is formatted to be sent to convection as a POST request. Since this artist name is used purely for display purposes on the "Describe your work" page, it seemed okay to keep them separate and not have to worry about merging in the `artist` into the `inputs` later.